seagoat/cli.py

```diff
@@ -175,6 +174,14 @@ def seagoat(
         )
         sys.exit(ExitCode.SERVER_NOT_RUNNING)
 
+    try:
+        warn_if_update_available()
+    except requests.exceptions.ConnectionError:
+        click.echo(
+            "Could not check for updates. "
+            "Please check your internet connection and try again."
```

Comment:
```suggestion
```

I would remove this line, I think it's excessive/distracting because this does not imply that the user explicitly tried to check for updates, so probably they don't need to try again

Reply:
instead, you could change `"Could not check for updates. "` 

to something like `"Could not check for updates because the pypi.org API is not accessible"

Reply:
Agree